spec/rubocop/cop/sorbet/multiple_t_enum_values_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe(RuboCop::Cop::Sorbet::MultipleTEnumValues, :config) do
+  it "registers an offense when creating a T::Enum with no values" do
+    expect_offense(<<~RUBY)
+      class MyEnum < T::Enum
+      ^^^^^^^^^^^^^^^^^^^^^^ `T::Enum` should have at least two values.
+        enums do
+        end
+      end
+    RUBY
+  end
+
+  it "registers an offense when creating a T::Enum with one value" do
+    expect_offense(<<~RUBY)
+      class MyEnum < T::Enum
+      ^^^^^^^^^^^^^^^^^^^^^^ `T::Enum` should have at least two values.
+        enums do
+          A = new
+        end
+      end
+    RUBY
+  end
+
+  it "does not register an offense when creating a T::Enum with two values" do
+    expect_no_offenses(<<~RUBY)
+      class MyEnum < T::Enum
+        enums do
+          A = new
+          B = new
+        end
+      end
+    RUBY
+  end
+
+  it "registers an offense for a nested T::Enum" do
+    expect_offense(<<~RUBY)
+      class MyEnum < T::Enum
+        class NestedEnum < T::Enum
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^ `T::Enum` should have at least two values.
+          enums do
+            A = B
+          end
+        end
+      end
+    RUBY
+  end
+
+  it "registers an offense for outer T::Enum" do
+    expect_offense(<<~RUBY)
+      class MyEnum < T::Enum
+      ^^^^^^^^^^^^^^^^^^^^^^ `T::Enum` should have at least two values.
+        class NestedEnum < T::Enum; end
+
+        enums do
+          A = B
+        end
+      end
+    RUBY
+  end
+end
```

Comment:
Would this break since we never reset the scope when entering a new class?

```rb
class MyEnum < T::Enum
  class Foo
    enums do; end
  end
end
```

Reply:
Hmm we may also want to check for this?

```rb
class MyEnum < T::Enum
end
```

In which case we would need to keep the offense on the class rather than the `enums` block.

Reply:
For your first example, I think that would just be a method error because `enums` wouldn't exist on the `Foo` class?

Good catch on the second example.

Reply:
Added a test for the second example.